tests/acceptance/10_files/02_maintain/absolute_link_and_no_move_obstructions.cf

```diff
@@ -44,8 +44,8 @@ bundle agent test
 
   files:
       "$(G.testfile)"
-      classes => init_if_failed("test_ok"),
-      link_from => test_link;
+          classes => init_if_failed("test_ok"),
+          link_from => test_link;
```

Comment:
Sooo I just have to edit `reindent.pl` to change the stinky indentation rules? It's perl but I think I might have to do that...